Floofbot/Services/EventLoggerService.cs

```diff
@@ -389,11 +389,18 @@ public Task UserLeft(IGuildUser user)
                         return;
 
                     var embed = new EmbedBuilder();
-
                     embed.WithTitle($"❌ User Left | {user.Username}#{user.Discriminator}")
                          .WithColor(Color.Red)
-                         .WithDescription($"{user.Mention} | ``{user.Id}``")
-                         .WithFooter($"user_leave user_joinlog {user.Id}")
+                         .WithDescription($"{user.Mention} | ``{user.Id}``");
+                    if (user.JoinedAt != null)
+                    {
+                        DateTimeOffset userJoined = ((DateTimeOffset)user.JoinedAt);
+                        TimeSpan interval = DateTime.UtcNow - userJoined.DateTime;
```

Comment:
We're using UtcNow here?
I thought we were going to use the local server time which would just be DateTime.Now?

Reply:
The value returned from the API is UTC, so I need to use UtcNow to get an accurate interval.